### PR TITLE
Created a new create account html, changes to app.py funtion login and create account - Stephan

### PR DESCRIPTION
**Pull request by thedirtyham**

### CODE REVIEW

File: static/jquery/AUTHORS.txt [hunk 1]
```diff
@@ -0,0 +1,295 @@
+Authors ordered by first contribution.
```
Comment (algorithmike) [inline static/jquery/AUTHORS.txt:1]: Where are these from? Are these real email addresses?

File: app.py [hunk 2]
```diff
@@ -1,27 +1,65 @@
...
+from flask_wtf import Form
+from wtforms import TextField, TextAreaField, validators, StringField, SubmitField, csrf
+from wtforms.validators import DataRequired
 from slackclient import SlackClient
+from hashlib import md5
+
 
 app = flask.Flask(__name__)
 
-@app.route('/base')
```
Comment (algorithmike) [inline app.py:6]: What happened to base?

**Comment by thedirtyham** [comment]
Everything works the same. I downloaded jquery. I don't think tho are real
emails.

File: static/jquery/AUTHORS.txt (hunk 1 above)
Comment (algorithmike) [reply static/jquery/AUTHORS.txt:1]: @thedirtyham

File: app.py (hunk 2 above)
Comment (algorithmike) [reply app.py:6]: @eliasargandara @Salv95 Input?

**Pull request merged**